src/Forms/PluginsGet.cs

```diff
@@ -86,16 +95,9 @@ private void DownloadPluginMetadataInfos()
                 Refresh();
                 ShowInstalledPlugins();
                 string url = GetPluginXmlFileUrl();
-                var wc = new WebClient
-                {
-                    Proxy = Utilities.GetProxy(),
-                    Encoding = System.Text.Encoding.UTF8,
-                    CachePolicy = new RequestCachePolicy(RequestCacheLevel.NoCacheNoStore)

```

Comment:
I think this wasn't working properly, I updated the plugin4.xml file it it took couple minutes to the changes to show up in PluginGet form